wallet/constants.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
   * the "5820" part is the CBOR prefix for a hex string
   */
   TX_SIGN_MESSAGE_PREFIX: '011a2d964a095820',
+  CARDANO_KEY_DERIVATION_MODE: 1,
```

Comment:
I would add some comment with link to some doc (if there is some) or some explanation of what possible values are and what is their meaning